crates/noirc_frontend/src/hir/def_map/mod.rs

```diff
@@ -89,7 +89,7 @@ impl CrateDefMap {
         let ast = parse_file(&mut context.file_manager, root_file_id, errors);
 
         #[cfg(feature = "aztec")]
-        let ast = aztec_library::transform(ast);
+        let ast = aztec_library::transform(ast, &crate_id, context, errors);
```

Comment:
Is this failing on master then?

Reply:
It should be, but no tests run with this as it is flagged